File: csL2/Domain/PersoanaOficiu.cs
namespace lab2_mpp.domain;

public class PersoanaOficiu : Entity<int>
{
    public String OrasOficiu { get; set; }
    public String Username { get; set; }
    public String Password { get; set; }

    public PersoanaOficiu(String orasOficiu, String username, String password)
    {
        OrasOficiu = orasOficiu;
        Username = username;
        Password = password;
    }
    
    public override String ToString()
    => $"PersoanaOficiu {{ orasOficiu= {OrasOficiu}, user= {Username}, pass={Password} }}";
}

File: javaL2/src/main/java/ro/mpp2024/domain/PersoanaOficiu.java
package ro.mpp2024.domain;

public class PersoanaOficiu extends Entity<Integer> {
    private String orasOficiu;
    private String username;
    private String password;

    public PersoanaOficiu(String orasOficiu, String username, String password) {
        this.orasOficiu = orasOficiu;
        this.username = username;
        this.password = password;
    }
    public String getOrasOficiu() {
        return orasOficiu;
    }
    public void setOrasOficiu(String orasOficiu) {
        this.orasOficiu = orasOficiu;
    }
    public String getUsername() {
        return username;
    }
    public void setUsername(String username) {
        this.username = username;
    }
    public String getPassword() {
        return password;
    }
    public void setPassword(String password) {
        this.password = password;
    }

}
